main.cpp

```diff
@@ -1,19 +1,51 @@
-#include <iostream>
-#include <cstdlib>
-
-int getNumber( int min, int count )
-{
-    return min + rand() % count ;
-}
-
+#include <iostream>  
+#include<Windows.h>
+#include<math.h>
```

Comment:
Старайтесь использовать один стиль.
```suggestion
#include <iostream>  

#include <Windows.h>
#include <math.h>
```